src/client/pages/OfferNew/Introduction/Sidebar/DetailsModal/utils.ts

```diff
@@ -554,7 +554,11 @@ export const getInitialSwedishHouseValues = (
     numberOfBathrooms: details.numberOfBathrooms,
     yearOfConstruction: details.yearOfConstruction,
     isSubleted: details.isSubleted,
-    extraBuildings: details.extraBuildings,
+    extraBuildings: details.extraBuildings.map((building) => ({
```

Comment:
Perhaps - to make it even more explicit why it's needed?

```js
    extraBuildings: details.extraBuildings.map<ExtraBuildingInput>((building) => ({
```

Reply:
Shouldn't it be an array?
```typescript
extraBuildings: details.extraBuildings.map<ExtraBuildingInput[]>((building) => ({
```

Reply:
But I like being more explicit 👌